### DISCUSSION TITLE
Semantic convension for logged in username?

### DISCUSSION THREAD

**Discussion by stnor** [open-telemetry/opentelemetry-specification#1706] [Q&A] [answered]
Hi,
Is there any thoughts on adding semantic conventions to collect end-user attributes (eg username, roles) to the spec?
I can't find anything at the moment?

Most commercial tools for looking at traces such as AppD, AWS X-Ray etc that I've used have at least username as a filter dimension in their ui.

**Comment by Oberon00** [comment, marked as answer]
These attributes are hidden in span-general.md: https://github.com/open-telemetry/opentelemetry-specification/blob/v1.3.0/specification/trace/semantic_conventions/span-general.md#general-identity-attributes

**Reply by stnor** [reply]
Thanks!
